packages/shared/src/introspectMachine.ts

```diff
@@ -272,6 +276,10 @@ export const introspectMachine = (machine: XState.StateNode) => {
         actions.addEventToItem(action.type, source, node.path);
       });
     });
+
+    node.onExit.forEach((action) => {
+      actions.addEventToItem(action.type, "xstate.stop", node.path);
```

Comment:
Since we are adding `xstate.init` to `internalEvents` of the generated typegen info, we should do the same for `xstate.stop`.

Reply:
It does that already - anything with `xstate.*` is added.

Reply:
ah, got it - didn't know that is automated this way :p